tests/features/asset_distribution/add_asset_distribution.feature

```diff
@@ -5,20 +5,19 @@ Feature: Add asset distribution through the UI
   I need to be able to add "Asset distribution" RDF entities through the UI.
 
   Scenario: "Add distribution" button should only be shown to moderators.
-    Given the following collection:
-      | title | Asset Distribution Test |
-      | logo  | logo.png                |
-    And the following solution:
-      | title       | Asset random name                |
+    Given the following solution:
+      | title       | Solution random x name           |
       | description | Some reusable random description |
-      | collection  | Asset Distribution Test          |
+    And the following collection:
+      | title      | Asset Distribution Test |
+      | logo       | logo.png                |
+      | affiliates | Solution random x name  |
     And the following asset release:
       | title         | Asset release random name        |
       | description   | Some reusable random description |
-      | is version of | Asset random name                |
-    When I am logged in as a "moderator"
+      | is version of | Solution random x name           |
+    When I am logged in as a "facilitator" of the "Solution random x name" solution
```

Comment:
This is great stuff, we are no longer limited to being a moderator to administer the site.